src/constant/index.js

```diff
@@ -81,7 +81,8 @@ export const LIKECOIN_NFT_API_WALLET = IS_TESTNET
 export const LIKECOIN_NFT_BOOK_ITEMS = IS_TESTNET
   ? [
       'likenft1fsf4x0tsm8gj8x0adkcce68es2xt5gwk9y7zjf06wwh92t27s99q3ktwpn',
-      'likenft1wyk82qhawxh5q27ruc0dj0053unmddk3mswwvuhf6ste6v93e7rqtvl0fy',
+      'likenft1xj9jkxcernd2fzd6597tee3v57hzy8r4tzpmr7hu5mqvzah7ztaqua9yvh',
+      // 'likenft1wyk82qhawxh5q27ruc0dj0053unmddk3mswwvuhf6ste6v93e7rqtvl0fy',
```

Comment:
What is this for?

Reply:
Now all featured books are out of stock, for QA convenience.

Reply:
I think just remove the commented item would do